charms/istio-pilot/metadata.yaml

```diff
@@ -22,8 +22,9 @@ provides:
     versions: [v1]
   ingress:
     interface: ingress
-    schema: https://raw.githubusercontent.com/canonical/operator-schemas/master/ingress.yaml
-    versions: [v1, v2]
+    #schema: https://raw.githubusercontent.com/canonical/operator-schemas/master/ingress.yaml
+    schema: https://raw.githubusercontent.com/canonical/operator-schemas/1ed74c640bc289b71f261cda67177ee5209a1562/ingress.yaml
+    versions: [v1, v2, v3]
   ingress-auth:
```

Comment:
What does this do? I cannot figure it out from the code :D

Reply:
The `ingress` interface and the istio-pilot operator charm were designed using the [Serialized Data Interface (SDI)](https://github.com/canonical/serialized-data-interface) library, and it uses these non-canonical fields to reference the schema and note which versions are supported by the charm.